Makefile

```diff
@@ -10,5 +15,18 @@ jar:
 release:
 	mvn -B release:prepare release:clean
 
-full:
-	mvn clean checkstyle:check cobertura:cobertura javadoc:jar package
+coverage:
+	mvn clean cobertura:cobertura
+
+doc:
+	mvn clean javadoc:javadoc
+
+see-coverage: coverage
+	cd target/site/cobertura; python -m SimpleHTTPServer
```

Comment:
I think since the Makefile itself is specific to our dev usage (most people would just use mvn), I am putting stuff in there that makes it easy to do dev things. Seeing javadocs and code coverage seems like something you'd do often. I certainly did for code coverage.